subtests/docker_cli/run_volumes/run_volumes.py

```diff
@@ -187,16 +200,8 @@ def initialize(self):
         super(volumes_rw, self).initialize()
         host_paths = get_as_list(self.config['host_paths'])
         cntr_paths = get_as_list(self.config['cntr_paths'])
```

Comment:
Would it make sense to add `, omit_empty=True` above? That would give a more helpful message when config settings are empty.